src/NetscriptFunctions/Singularity.ts

```diff
@@ -1300,7 +1300,15 @@ export function NetscriptSingularity(
         throw helper.makeRuntimeErrorMsg("commitCrime", `Invalid crime: '${crimeRoughName}'`);
       }
       workerScript.log("commitCrime", () => `Attempting to commit ${crime.name}...`);
-      return crime.commit(Router, player, 1, workerScript);
+      crime.commit(Router, player, 1, workerScript);
```

Comment:
This is not taking into consideration if there was focus before hand, so every time the commitCrime triggers it'll send you back to the terminal even if you aren't focusing or were looking at a different screen.

For instance see line 1278 on how it's implemented for working on a program.